### DISCUSSION TITLE
mysql/redis-shuju-yizhixing

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#178] [Announcements]
# mysql/redis-shuju-yizhixing

整理：沉默王二，戳转载链接，作者：楼仔，戳原文链接。 如何保障 MySQL 和 Redis 的数据一致性？这个问题很早之前我就遇到过，但是一直没有仔细去研究，上个月看了极客的课程，有一篇文章专门有过讲解，刚好有粉丝也问我这个问题，所以感觉有必要单独出一篇。

https://tobebetterjavaer.com/mysql/redis-shuju-yizhixing.html

**Comment by Che-cy** [comment]
为啥我看不到图片呢

**Comment by qi7chen** [comment]
图做的很不错，请问用什么工具制作的呢
